WordPress/Classes/Networking/Pinghub.swift

```diff
@@ -77,7 +86,14 @@ public class PinghubClient {
             guard let client = self else {
                 return
             }
-            client.delegate?.pinghubDidDisconnect(client, error: error)
+            let filteredError: NSError? = error.flatMap({ error in
+                // Filter out normal disconnects that we initiated
+                if error.domain == WebSocket.ErrorDomain && error.code == Int(WebSocket.CloseCode.normal.rawValue) {
+                    return nil
+                }
+                return error
+            })
```

Comment:
Do we need to filter the error, for a specific reason?. (It _feels_ like... it's the delegate the one who should check on the error condition, instead of us, filtering it)